internal/pkg/transformers/tee.go

```diff
@@ -35,6 +35,8 @@ Any of the output-format command-line flags (see mlr -h). Example: using
   mlr --icsv --opprint put '...' then tee --ojson ./mytap.dat then stats1 ...
 the input is CSV, the output is pretty-print tabular, but the tee-file output
 is written in JSON format.
+
```

Comment:
At the beginning, I saw the "-h" option multiple times in the same text, so I thought it was superfluous. Later I realized that the text was just the concatenation of the individual help commands, hence each one have its own "-h" flag.
It could make some sense to remove those multiple lines from the compendium, but that meant removing from the individual commands, which was not the expected outcome.
Unfortunately, I realized after pushing the commit. Fortunately, it can be reverted :-)